man/fread.Rd

```diff
@@ -15,14 +15,14 @@ skip=0L, select=NULL, drop=NULL, colClasses=NULL,
 integer64=getOption("datatable.integer64"),         # default: "integer64"
 dec=if (sep!=".") "." else ",", col.names, 
 check.names=FALSE, encoding="unknown", quote="\"", 
-strip.white=TRUE, fill=FALSE, blank.lines.skip=FALSE, key=NULL, 
+strip.white = ifelse(sep == "\n", FALSE, TRUE), fill=FALSE, blank.lines.skip=FALSE, key=NULL, 
```

Comment:
`ifelse` could be more useful for vectorized input, here `strip.white = sep!="\n"` or `strip.white = !identical(sep,"\n")` should be sufficient, isn't it? the latter one if we allow 0 length `sep` argument.